### ISSUE TITLE
[Bug]: img-to-img Unit 1 wont let me select Control Type, mouse icon turn into a prohibition icon

### ISSUE THREAD

**Issue by fifskank** [Mikubill/sd-webui-controlnet#2593]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

in imgtoimg Unit 0 is enabled and i can select Control Type, but when i got to Unit 1, the mouse turns into prohibition icon in Control Type and wont let me select any

### Steps to reproduce the problem

1. Go to .... imgtoimg Enable Unit 0, select Control Type, then go to Unit 1 and Enable it and try to select a Control Type
2. Press .... the Control Type section but mouse turns in prohibition icon
3. ...

### What should have happened?

enable Unit 0 and select Control Type, Tile/Blur, then go to Unit 1 and select Control Type Canny,

### Commit where the problem happens

multi controlnet units only works Unit 0, Unit 1 cant select Control Type

### What browsers do you use to access the UI ?

Mozilla Firefox

### Command Line Arguments

```Shell
im using directml version for AMD
```

### List of enabled extensions

only ControlNet

### Console logs

```Shell
2024-01-28 00:41:54,190 - ControlNet - INFO - unit_separate = False, style_align = False02/153 [11:15<05:23,  6.34s/it]
2024-01-28 00:41:54,191 - ControlNet - WARNING - [tile_resample.threshold_a] Invalid value(-1), using default value 1.0.
2024-01-28 00:41:54,192 - ControlNet - INFO - Loading model from cache: control_v11f1e_sd15_tile [a371b31b]
2024-01-28 00:41:54,232 - ControlNet - INFO - Using preprocessor: tile_resample
2024-01-28 00:41:54,232 - ControlNet - INFO - preprocessor resolution = -1
2024-01-28 00:41:54,262 - ControlNet - INFO - ControlNet Hooked - Time = 0.07406759262084961
100%|██████████████████████████████████████████████████████████████████████████████████| 17/17 [01:48<00:00,  6.37s/it]
2024-01-28 00:43:48,410 - ControlNet - INFO - unit_separate = False, style_align = False19/153 [13:09<03:35,  6.34s/it]
2024-01-28 00:43:48,410 - ControlNet - WARNING - [tile_resample.threshold_a] Invalid value(-1), using default value 1.0.
2024-01-28 00:43:48,411 - ControlNet - INFO - Loading model from cache: control_v11f1e_sd15_tile [a371b31b]
2024-01-28 00:43:48,455 - ControlNet - INFO - Using preprocessor: tile_resample
2024-01-28 00:43:48,455 - ControlNet - INFO - preprocessor resolution = -1
2024-01-28 00:43:48,496 - ControlNet - INFO - ControlNet Hooked - Time = 0.09008121490478516
100%|██████████████████████████████████████████████████████████████████████████████████| 17/17 [01:48<00:00,  6.37s/it]
2024-01-28 00:45:42,693 - ControlNet - INFO - unit_separate = False, style_align = False36/153 [15:04<01:47,  6.34s/it]
2024-01-28 00:45:42,694 - ControlNet - WARNING - [tile_resample.threshold_a] Invalid value(-1), using default value 1.0.
2024-01-28 00:45:42,695 - ControlNet - INFO - Loading model from cache: control_v11f1e_sd15_tile [a371b31b]
2024-01-28 00:45:42,740 - ControlNet - INFO - Using preprocessor: tile_resample
2024-01-28 00:45:42,741 - ControlNet - INFO - preprocessor resolution = -1
2024-01-28 00:45:42,782 - ControlNet - INFO - ControlNet Hooked - Time = 0.09308433532714844
```

### Additional information

cant selet Control Type on Unit 1 nor Unit 2

**Comment by YousufSSyed** [comment]
Also having this issue

**Comment by FruitPigFoot** [comment]
me either

**Comment by huchenlei** [comment]
Please set `--controlnet-loglevel DEBUG` in your launch script and attach full console logs. Thanks!

**Comment by huchenlei** [comment]
Also I need a more reliable way of reproduction, The issue only happens to me once when we had bugs on https://github.com/Mikubill/sd-webui-controlnet/pull/2595.

**Comment by MrRedMinion** [comment]
I have been hitting this issue as well and it occurs right away as soon as you start the webui, if you try to select control unit 1, the icon is blocked. Version 1.1.431 (84ef2056737dab2c63e708464c547926168b1088) is the last known commit I was able to get it to work.

--controlnet-loglevel DEBUG doesn't show any new logs on the console unfortunately. I am suspecting it is a UI problem.

I am using sdnext, I was going to try automatic1111 and fresh install of sdnext to see if i can replicate. will post my findings

**Comment by huchenlei** [comment]
The log I am expecting is:
```
2024-01-29 00:06:47,655 - ControlNet - DEBUG - A1111 initialized 14/17.
2024-01-29 00:06:47,673 - ControlNet - DEBUG - Setting img2img_width.
2024-01-29 00:06:47,675 - ControlNet - DEBUG - A1111 initialized 15/17.
2024-01-29 00:06:47,676 - ControlNet - DEBUG - Setting img2img_height.
2024-01-29 00:06:47,677 - ControlNet - DEBUG - A1111 initialized 16/17.
2024-01-29 00:06:47,776 - AnimateDiff - INFO - Hacking i2i-batch.
2024-01-29 00:06:47,847 - ControlNet - DEBUG - uigroup ran in: 0.064 sec
2024-01-29 00:06:47,915 - ControlNet - DEBUG - uigroup ran in: 0.066 sec
2024-01-29 00:06:47,982 - ControlNet - DEBUG - uigroup ran in: 0.064 sec
2024-01-29 00:06:48,045 - ControlNet - DEBUG - uigroup ran in: 0.061 sec
2024-01-29 00:06:48,112 - ControlNet - DEBUG - uigroup ran in: 0.064 sec
2024-01-29 00:06:48,181 - ControlNet - DEBUG - uigroup ran in: 0.066 sec
2024-01-29 00:06:48,245 - ControlNet - DEBUG - uigroup ran in: 0.062 sec
2024-01-29 00:06:48,319 - ControlNet - DEBUG - uigroup ran in: 0.072 sec
2024-01-29 00:06:48,382 - ControlNet - DEBUG - uigroup ran in: 0.061 sec
2024-01-29 00:06:48,456 - ControlNet - DEBUG - uigroup ran in: 0.072 sec
2024-01-29 00:06:48,550 - ControlNet - INFO - ControlNet UI callback registered. <--- This log must appear.
```

You should see it in your command line console, not the browser console.

**Comment by MrRedMinion** [comment]
after a fresh install of sdnext the issue can't be replicated.

**Comment by MrRedMinion** [comment]
is there a different way to pass --controlnet-loglevel DEBUG through sdnext? I can't get any logs from it. ill post on that repo if you are not aware of any difference

**Comment by fifskank** [comment]
im doin this ill get back ASAP !!!! thank you

> Please set `--controlnet-loglevel DEBUG` in your launch script and attach full console logs. Thanks!

**Comment by fifskank** [comment]
![debug_](https://github.com/Mikubill/sd-webui-controlnet/assets/132480167/fc01f2b5-13a1-4cce-8409-4e573707accb)

there you go i hope this helps, cause having multiple processor for controlnet is what makes it so great !!!!!
![debug_](https://github.com/Mikubill/sd-webui-controlnet/assets/132480167/f35b17b0-394c-4eb6-88fa-3cc016cce33b)

**Comment by huchenlei** [comment]
Wow interesting, only 9/17 A1111 components expected by ControlNet are available. I see you are running A1111 1.5.1. Can you upgradte your A1111 to at least 1.6? We really cannot guarantee that latest version ControlNet always compatible with legacy version of A1111.

**Comment by huchenlei** [comment]
After some investigation, the `enable_hr` checkbox element was not available in 1.5.0 A1111. We can obviously make that component optional as well, but I would still recommend you upgrade your A1111 version, as this extension will be developed and tested against latest stable version of A1111 (Currently 1.7.0).
